openedx/core/djangoapps/course_roles/permissions.py

```diff
@@ -0,0 +1,97 @@
+"""
+Bridgekeeper permissions for course roles
+"""
+
+from bridgekeeper import perms
+
+from lms.djangoapps.courseware.rules import HasRolesRule
+from openedx.core.djangoapps.course_roles.data import CourseRolesPermission
+from openedx.core.djangoapps.course_roles.rules import HasPermissionRule, HasForumsRolesRule
+
+
+perms[f'course_roles.{CourseRolesPermission.MANAGE_CONTENT.value.name}'] = (
+    HasRolesRule('staff', 'instructor')
```

Comment:
Isn't the role to permissions mapping already encoded into the data tables with this PR? Why do we still have to check against these roles here?

Reply:
The roles checks are being included to ensure that a user who already has a `student_courseaccessrole` continues to have the proper access. The current plan is to add permissions checks to ensure the definitions of the permissions match the access granted by the code. This code ensures that a user with a `student_courseaccessrole` will retain the access their role should have even if their role has not yet been migrated to `course_roles`.

Existing data is planned to be migrated one role at a time. Once all existing `student_courseaccessrole` role data has been migrated to the `course_roles` schema, it will no longer be necessary to include the roles checks in this permissions file. The role checks for a role will be removed from this file as each role is migrated.